### PR TITLE
Integrate with supporter experience api

### PR DESCRIPTION
**Pull request by rahilb**

### CODE REVIEW

**Comment by rahilb** [comment]
more javascript for @walaura 🙈

**Comment by rahilb** [comment]
This has been tested in code with some users that Jacob was kind enough to set up, see screenshots:  

screen shot 2018-05-23 at 15 37 25
screen shot 2018-05-23 at 15 32 06

File: cloudformation/retention.yaml [hunk 1]
```diff
@@ -11,21 +11,27 @@ Parameters:
...
+    NoEcho: true
+  EncryptionKeyArn:
     Type: String
     Description: ARN of the KMS key, used to encrypt environment variables passed to lambdas and step functions
     Default: arn:aws:kms:eu-west-1:942464564246:key/a39e1922-4a2c-4c73-ae00-8a97ef905428
 Mappings:
   StageVariables:
     PROD:
       IdapiRoot: idapi.theguardian.com
+      SupporterExRoot: https://wb5tif9s0e.execute-api.eu-west-1.amazonaws.com/PROD
```
Comment (jfsoul) [inline cloudformation/retention.yaml]: are these secret? (or at least semi-secret)

Comment (rahilb) [reply cloudformation/retention.yaml]: I don't think so, an API key is required to call the endpoint and I think this is being replaced with a proper subdomain on gu.com, maybe @jacobwinch  can advise?

Comment (jacobwinch) [reply cloudformation/retention.yaml]: Yep, I've been sorting out CNAMEs this afternoon. I'll send over nice(r) URLs and API keys (via zerobin) for both environments tomorrow.

**Pull request merged**